compiler/coreSyn/CoreLint.hs

```diff
@@ -761,6 +761,7 @@ lintCoreExpr (Let (NonRec tv (Type ty)) body)
 lintCoreExpr (Let (NonRec bndr rhs) body)
   | isId bndr
   = do  { let_ue <- lintSingleBinding NotTopLevel NonRecursive (bndr,rhs)
+          -- See Note [Multiplicity of let binders] in Var
```

Comment:
Link also in the `let rec` case.

(Which, I gather, was previously broken altogether?)